lewis_emulators/wbvalve/interfaces/stream_interface.py

```diff
@@ -0,0 +1,47 @@
+from lewis.adapters.stream import StreamInterface, Cmd
+from lewis.core.logging import has_log
+
+from lewis_emulators.utils.command_builder import CmdBuilder
+from lewis_emulators.utils.replies import conditional_reply
+
+if_connected = conditional_reply("connected")
+
+@has_log
+class WbvalveStreamInterface(StreamInterface):
+
+    in_terminator = '\n'
+    out_terminator = '\n'
+
+    def __init__(self):
+
+        super(WbvalveStreamInterface, self).__init__()
+        self.commands = {
+            CmdBuilder(self.get_position).escape("st").build(),
+            CmdBuilder(self.set_position).escape("wb").int().escape("on").build()
```

Comment:
It won't catch you out here as the commands don't overlap, but in general you need to put `.eos()` as the last item before `.build()`. This anchors the generated regex to the end of the string. This can catch you out if you have commands that partially overlap.